src/main/java/com/fnp/Serialization/serializationresource/Company.java

```diff
@@ -0,0 +1,57 @@
+package serializationresource;
+
+import java.io.Serializable;
+
+public class Company implements Serializable{	
+	
+	public static final long serialversionUID = 7642873L;
+	private transient int id;
+	private	String name;
+	private transient String location;
+	private static String dept;
```

Comment:
you can also add  javadoc for each field, method and class level @saranshvijay ignore for now..take care in later commits.